aidial_assistant/application/assistant_application.py

```diff
@@ -99,23 +99,24 @@ async def chat_completion(
             buffer_size=self.args.chat_conf.buffer_size,
         )
 
-        addons: list[str] = (
-            [addon.url for addon in request.addons] if request.addons else []  # type: ignore
+        token_source = AddonTokenSource(
+            request.headers,
+            [addon.url for addon in request.addons] if request.addons else [],  # type: ignore
```

Comment:
Please ignore typechecking ignores at all costs. Typechecker is a fried, not enemy.

Make `_validate_addons` return the list of addons whose `url` is required.
Or fix the SDK to make `url` required.